app-android-xamarin/Moodis/Moodis/Feature/SignIn/SignInActivity.cs

```diff
@@ -53,6 +54,11 @@ protected override void OnActivityResult(int requestCode, [GeneratedEnum] Result
                 SetResult(Result.Ok, new Intent().PutExtra(EXTRA_SIGNED_IN, true));
                 Finish();
             }
+            else if (resultCode == Result.Ok && requestCode == REQUEST_CODE_REGISTER_FACE)
```

Comment:
Why was this added?

Reply:
If user signs in normaly he is prompt to update face. After face is updated he is logged in like normal..